src/server/routes.js

```diff
@@ -39,7 +39,9 @@ export default function (app) {
   app.post('/alexa/launch', handleLaunch);
   app.post('/alexa/estimate', AlexaGetEstimate);
 
-  // ===== internal routes ===== //
+  app.post('/alexa/launch/:alexaUserId', alexa.handleLaunch);
```

Comment:
@JasonArkens17 notice how the travis build failed with the following error: `ReferenceError: alexa is not defined`

not sure why it says `alexa.handleLaunch` on this line. can you take a look? i'll review this PR after hearing back on this